docs/synthetic-tokens/explainer.md

```diff
@@ -38,7 +38,7 @@ This smart contract can be deployed using UMA’s synthetic token contract templ
 - token expiration timestamp
```

Comment:
I think the `Priceless Synthetic Tokens` section should explicitly clarify that the rest of the page is talking about one particular flavor of synthetic tokens, the Expiring MultiParty

Reply:
A few updates to the layout: 

-Within "Overview" section, lets rename "Synthetic tokens" to "Priceless contracts". This lines up better with our website and this way we don't have two sections labeled "Synthetic tokens". 

-Within "Overview" rename "Architecture Overview" to "UMA Overview" (it isn't really an architecture overview)

Reply:
-Within section "Overviews" rename the top header that says "Overviews" to "Understanding UMA"

Reply:
Sections renamed - although I left architecture overview as is.

Also addressed Nick's comments by adding some clarification that the EMP is only one example of a type of synthetic token that could be built on UMA.